docs/getting_started.rst

```diff
@@ -0,0 +1,394 @@
+
+    # Show genotype calls with domain-specific display logic
+    sg.display_genotypes(ds, max_variants=8, max_samples=8)
+
+    # A naive version of the above is also possible using only Xarray/Pandas and
+    # illustrates the flexibility that comes from being able to transition into
+    # and out of array/dataframe representations easily
+    (ds.call_genotype[:5, :5].to_series()
+        .unstack().where(lambda df: df >= 0, None).fillna('.')
+        .astype(str).apply('/'.join, axis=1).unstack())
+
+    # Show call rate distribution for each variant using Pandas
+    df = ~ds.call_genotype_mask.to_dataframe()
+    df.head(5)
+
+    call_rates = df.groupby('variants').mean()
+    call_rates
+
+    @savefig call_rate_example.png width=6in height=3in
+    call_rates.plot(kind='hist', bins=24, title='Call Rate Distribution', figsize=(6, 3))
+
+This last example alludes to representations of missing data that are explained further in :ref:`missing_data`.
+
+Genetic methods
+~~~~~~~~~~~~~~~
+
+Genetic methods in sgkit are nearly always applied to individual ``Dataset`` objects.  For a full list of
+available methods, see :ref:`api_methods`.
+
+In this example, the ``variant_stats`` method is applied to a dataset to compute a number of statistics
+across samples for each individual variant:
+
+.. ipython:: python
+
+    import sgkit as sg
+    ds = sg.simulate_genotype_call_dataset(n_variant=100, n_sample=50, missing_pct=.1)
+    sg.variant_stats(ds, merge=False)
+
+There are two ways that the results of every function are handled -- either they are merged with the provided
+dataset or they are returned in a separate dataset.  See :ref:`dataset_merge` below for more details.
+
+.. _dataset_merge:
+
+Dataset merge behavior
+~~~~~~~~~~~~~~~~~~~~~~
+
+Generally, method functions in sgkit compute some new variables based on the
+input dataset, then return a new output dataset that consists of the input
+dataset plus the new computed variables. The input dataset is unchanged.
+
+This behavior can be controlled using the ``merge`` parameter. If set to ``True``
+(the default), then the function will merge the input dataset and the computed
+output variables into a single dataset. Output variables will overwrite any
+input variables with the same name, and a warning will be issued in this case.
+If ``False``, the function will return only the computed output variables.
+
+Examples:
+
+.. ipython:: python
+    :okwarning:
+
+    import sgkit as sg
+    ds = sg.simulate_genotype_call_dataset(n_variant=100, n_sample=50, missing_pct=.1)
+    ds = ds[['variant_allele', 'call_genotype']]
+    ds
+
+    # By default, new variables are merged into a copy of the provided dataset
+    ds = sg.count_variant_alleles(ds)
+    ds
+
+    # If an existing variable would be re-defined, a warning is thrown
+    import warnings
+    ds = sg.count_variant_alleles(ds)
+    with warnings.catch_warnings(record=True) as w:
```

Comment:
This makes it look like users have to catch warnings, but you're having to that here to avoid ipython failing (I think). Not sure if there's a better way of doing it - one option would be to have the commented out code that would fail. Or add a comment being explaining that you don't need to explicitly catch warnings.

Reply:
I was trying to avoid the failure but more so I wanted a way to have it actually render somewhere.  I couldn't find a way to have the directive do that.  What's the problem with expecting users to catch warnings if they want to rather than having them print somewhere?  I'm struggling to word a comment for it.  Mind wording one for me and I'll drop it in?

Reply:
FYI haven't forgotten about this but I opened https://github.com/pystatgen/sgkit/issues/288 to track it.  I plan on working on https://github.com/pystatgen/sgkit/issues/287 soon so it may naturally go away.